src/SqlVirtualMachine/SqlVirtualMachine/SqlVirtualMachineGroup/Cmdlet/RemoveAzureSqlVMGroup.cs

```diff
@@ -0,0 +1,126 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.ResourceManager.Common.Tags;
+using Microsoft.Azure.Commands.SqlVirtualMachine.Common;
+using Microsoft.Azure.Commands.SqlVirtualMachine.SqlVirtualMachine.Model;
+using Microsoft.Rest.Azure;
+
+
+namespace Microsoft.Azure.Commands.SqlVirtualMachine.SqlVirtualMachine.Cmdlet
+{
+    /// <summary>
+    /// Defines Remove-AzSqlVMGroup cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVMGroup", DefaultParameterSetName = ParameterSet.ParameterList, ConfirmImpact = ConfirmImpact.Medium, SupportsShouldProcess = true)]
+    [OutputType(typeof(AzureSqlVMGroupModel))]
+    public class RemoveAzureSqlVMGroup : AzureSqlVMGroupCmdletBase
+    {
+        /// <summary>
+        /// Sql virtual machine group resource to be removed
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = ParameterSet.InputObject,
+            Position = 0,
+            ValueFromPipelineByPropertyName = true,
+            ValueFromPipeline = true,
+            HelpMessage = HelpMessages.InputObjectSqlVMGroup)]
+        [Alias("InputObject")]
+        [ValidateNotNullOrEmpty]
+        public AzureSqlVMGroupModel SqlVMGroup { get; set; }
+
+        /// <summary>
+        /// Resource id of the sql virtual machine group that will be removed
+        /// </summary>
+        [Parameter(Mandatory = true,
+            ParameterSetName = ParameterSet.ResourceId,
+            Position = 0,
+            HelpMessage = HelpMessages.SqlVMGroupResourceId)]
```

Comment:
@rocchiele please add `ValueFromPipelineByPropertyName = true` to this parameter